custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -16,7 +16,8 @@
   ],
   "server_environment": [
     "softlayer",
-    "icds"
+    "icds",
+    "icds-new"
```

Comment:
You should be able to remove this now.

Reply:
FWIW, it's consistent with most of other PRs.

Reply:
Then can you remove it from the other UCRs? we no longer  run an icds-new environment so I think it makes sense to not add new code that references it